test/Npgsql.Tests/ConnectionTests.cs

```diff
@@ -12,6 +12,7 @@
 
 namespace Npgsql.Tests
 {
+    [NonParallelizable]
```

Comment:
Because we are dealing with environment variables which are process wide. So setting them in some thread leads another to fail if it uses a password callback delegate.

Reply:
I don't think we should make the whole suite non-parallelizable just because of 1-2 tests. Not sure exactly how NUnit works, but if it's not possible to specify `[NonParallelizable]` on specific tests we can just move these out to a new tes suite.

Reply:
Moved this attribute to tests cases already.